src/Common/PuzzToolPage/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactChild } from 'react';
+import PuzzToolDocumentTitle from '../PuzzToolDocumentTitle';
+import './index.scss';
+
+interface Props {
+  children: ReactChild;
+  description?: ReactChild;
+  title: string;
+}
+
+function PuzzToolPage({children, description, title}: Props) {
+  function renderDescription() {
+    if (description) {
+      return (
+        <div className="PuzzToolPage-description">
+          {description}
+        </div>
+      )
+    }
+
+    return undefined;
```

Comment:
If there's no description, would an empty string be better than undefined here?  It seems like this would push effort on all callers to handle the undefined case.

Reply:
The whole point is that React naturally leaves out `undefined` elements. This prevents the rendering of empty divs that we don't want.